wordpress-mobile/gutenberg-mobile
# Stop scrolling inside the code block

This PR changes the PlainText component to block scrolling inside it and by consequence on any block that uses it, for example the Code block

GB PR: https://github.com/WordPress/gutenberg/pull/21115

To test:
 - Open the demo app
 - Scroll to the code block on the demo content or add one
 - Try to scroll inside the code block, check that all the post view scrolls and not the code block content only.

PR submission checklist:
